receiver/nginxreceiver/scraper.go

```diff
@@ -48,12 +50,54 @@ func newNginxScraper(
 func (r *nginxScraper) start(_ context.Context, host component.Host) error {
 	httpClient, err := r.cfg.ToClient(host.GetExtensions())
 	if err != nil {
+		r.logger.Error("nginx", zap.Error(err), zap.String("status_code", "INVALID_ARGUMENT"))
```

Comment:
Rather than writing our own strings, let's switch to using this package: https://github.com/grpc/grpc-go/blob/master/codes/codes.go

Continue to use `zap.String`, but specify the code using the package:  `zap.String("status_code", codes.InvalidArgument)`

Reply:
will need to use the .String method. zap.String("status_code", codes.InvalidArgument.String())